src/twisted/runner/procmon.py

```diff
@@ -71,14 +72,18 @@ class DummyTransport:
 class LineLogger(basic.LineReceiver):
 
     tag = None
+    stream = None
     delimiter = b'\n'
 
     def lineReceived(self, line):
         try:
             line = line.decode('utf-8')
         except UnicodeDecodeError:
             line = repr(line)
-        log.msg(u'[%s] %s' % (self.tag, line))
+        ProcessMonitor.log.info(u'[{tag}] {line}',
```

Comment:
I'm pretty open to changing the names on these fields, since they didn't have semantic meaning with the old logger.

Reply:
Also had a shower thought: Calling this on ProcessMonitor means that it isn't possible to override the logger on an instance level. It's a bigger refactor, but we could also replace the dummy transport with a way to get the data back to the protocol and ultimately the monitor instance.